noaa_test.go

```diff
@@ -87,3 +87,17 @@ func TestChicagoOffice(t *testing.T) {
 	}
 	t.Error("noaa.Office(\"LOT\") should return valid office information.")
 }
+
+func TestChicgoHourly(t *testing.T) {
+	hourly, err := noaa.HourlyForecast("41.837", "-87.685")
+	if err != nil {
+		t.Error("noaa.HourlyForecast() should return valid data for Chicago.")
+	}
+	if len(hourly.Periods) == 0 {
+		t.Error("expected at least one period")
+	} else {
+		for _, item := range hourly.Periods {
```

Comment:
Probably overhead we don't need in the test body (i.e. when running `go test -v`) unless there is some test/validation to be done on each element of the returned data set. (vs. printing them to the log.) This would be great for a new example in the README.md though.

Reply:
I assume you're referring to the `for _, item := range hourly.Periods`...

I only did that to validate that it actually fetched what I expected.  I will remove it.

Reply:
Hmmm... Godoc provides testable examples (re: https://go.dev/blog/examples), but this requires that the output is always the same, which can't happen directly with dynamic data such as fetching a weather forecast.  At best, one could use regular expressions to match against the output to provide validation and output the results of the regular expression, but that lacks a certain emotional impact to the output in this case.  But, I'll think on this; integrated, testable examples would be useful for demonstrating how one uses this library.